finesapp/src/main/kotlin/ua/anastasiia/finesapp/dto/Mappers.kt

```diff
@@ -0,0 +1,82 @@
+package ua.anastasiia.finesapp.dto
+
+import com.google.protobuf.util.Timestamps
+import com.google.type.LatLng
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.core.geo.GeoJsonPoint
+import ua.anastasiia.finesapp.commonmodels.car.Car
+import ua.anastasiia.finesapp.commonmodels.fine.Fine
+import ua.anastasiia.finesapp.commonmodels.trafficticket.TrafficTicket
+import ua.anastasiia.finesapp.commonmodels.violation.Violation
+import ua.anastasiia.finesapp.entity.MongoFine
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+
+fun MongoFine.toProto(): Fine =
+    Fine.newBuilder()
+        .setId(id?.toHexString() ?: ObjectId().toHexString())
```

Comment:
is it okay to return a random Id here? if your entity may not have Id consider making this parameter optional in proto

Reply:
removed this part: `?: ObjectId().toHexString()`. i found that in proto3, all fields are "optional" by default so does it make sense making it optional?

Reply:
not all fields are optional. if it is plain string, you have to define it optional